scene/audio/audio_stream_player.cpp

```diff
@@ -220,43 +126,19 @@ AudioStreamPlayer::MixTarget AudioStreamPlayer::get_mix_target() const {
 }
 
 void AudioStreamPlayer::_set_playing(bool p_enable) {
-	if (p_enable) {
-		play();
-	} else {
-		stop();
-	}
+	internal.set_playing(p_enable);
 }
 
 bool AudioStreamPlayer::_is_active() const {
```

Comment:
This method code was the same as `is_playing()`. I unified it to do `active.is_set()` as other players, idk if it makes difference.

Reply:
Hm… for AudioStreamPlayer, the meaning changes from "AudioServer has my playback registered and setup, and started playing" to "I started being processed and the AudioServer is about to start setting up my playback".

But no idea if this has any tangible repercussions. `_is_active` is a Node thing afaik

Reply:
Not sure if relevant, but does animation use this for syncing the start? Audio already starts earlier than other things. For reference: https://github.com/godotengine/godot/issues/82698#issuecomment-1871426859